SPECS/python-sortedcontainers/python-sortedcontainers.spec

```diff
@@ -0,0 +1,151 @@
+%global srcname sortedcontainers
+
+Summary:        Pure Python sorted container types
+Name:           python-%{srcname}
+Version:        2.4.0
+Release:        6%{?dist}
+Summary:        Pure Python sorted container types
+License:        ASL 2.0
+URL:            https://pypi.python.org/pypi/%{srcname}
+Source0:        https://github.com/grantjenks/python-sortedcontainers/archive/v%{version}.tar.gz#/%{srcname}-%{version}.tar.gz
```

Comment:
spec file is missing these tags:
Vendor:         Microsoft Corporation
Distribution:   Mariner